pnpbridge/src/adapters/src/shared/adapter_manifest.c

```diff
@@ -2,11 +2,12 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 //Pnp Adapter headers
-#include <pnpbridge.h>
+#include <pnpadapter_api.h>
 
 extern PNP_ADAPTER SerialPnpInterface;
 extern PNP_ADAPTER ModbusPnpInterface;
 extern PNP_ADAPTER MqttPnpInterface;
+extern PNP_ADAPTER EnvironmentSensorInterface;
```

Comment:
>EnvironmentSensorInterface [](start = 19, length = 26)

Please add a comment or rename to reflect this is a sample virtual sensor for illustrative purposes